.github/config.yml

```diff
@@ -0,0 +1,10 @@
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+    Congratulations on having your first pull request merged!
+    The Iris community are stoked that you've contributed and we all look forward to
+    your future work!
```

Comment:
@pelson nice 👍 I wonder whether we don't need this second exclamation mark though – it almost makes the cheer feel forced. I'd personally go for a smile 😄 here, but a full stop would also work.

Reply:
Good suggestion. Will test this out on a new repo (I'll grab you to come help me out if you don't mind) and then we can merge.